RoslynPluginGenerator/Program.cs

```diff
@@ -37,7 +38,9 @@ public static class Program
         private static int Main(string[] args)
         {
             ConsoleLogger logger = new ConsoleLogger();
-            Utilities.LogAssemblyVersion(typeof(Program).Assembly, UIResources.AssemblyDescription, logger);
+            Utilities.LogAssemblyVersion(typeof(Program).Assembly, UIResources.Program_AssemblyDescription, logger);
+            Utilities.LogAssemblyVersion(typeof(DiagnosticAnalyzer).Assembly, UIResources.Program_SupportedRoslynVersion, logger);
+            logger.LogInfo(UIResources.Program_SupportedSonarQubeVersions);
```

Comment:
Log the supported versions of Roslyn and SQ at startup to make it easier for the users to know what should work.